### PR TITLE
Do16 21 testing git features

### PR DESCRIPTION
**Pull request by xcarruth**

### CODE REVIEW

File: MVVMDemoNew.Tests/MVVMDemoNew.Tests.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
     <Platforms>AnyCPU;x64</Platforms>
+	<EnableWindowsTargeting>true</EnableWindowsTargeting>
```
Comment (AECOM-CarruthX) [inline MVVMDemoNew.Tests/MVVMDemoNew.Tests.csproj:11]: Test comment

File: MVVMDemoNew/Cart/CartView.xaml [hunk 2]
```diff
@@ -44,7 +44,7 @@
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <TextBlock Text="Cart" Margin="5" FontSize="22" />
-        <Button Content="Go Back To Catalog"
+        <Button Content="Catalog"
```
Comment (AECOM-CarruthX) [inline MVVMDemoNew/Cart/CartView.xaml]:
```suggestion
        <Button Content="Shopping Catalog"
```

File: MVVMDemoNew/Cart/CartViewModel.cs [hunk 3]
```diff
@@ -38,7 +38,7 @@ public CartViewModel()
         {
             CartItems = new ObservableCollection<ShoppingItem>()
             {
-                new ShoppingItem(new Guid(), "Navy Pants", "Navy is an terrible color for leg covers.", "pack://application:,,,/MVVMDemoNew;component/Images/navy pants.jpg", 1),
+                new ShoppingItem(new Guid(), "Navy Pants", "Navy is a horrible color for pants.", "pack://application:,,,/MVVMDemoNew;component/Images/navy pants.jpg", 1),
```
Comment (AECOM-CarruthX) [inline MVVMDemoNew/Cart/CartViewModel.cs]: Navy is a great color!

**Review by AECOM-CarruthX — CHANGES_REQUESTED** [review]
Completed review

File: MVVMDemoNew/Cart/CartViewModel.cs (hunk 3 above)
Comment (xcarruth) [reply MVVMDemoNew/Cart/CartViewModel.cs]: I think Navy is the worst. We should leave it as is.

File: MVVMDemoNew.Tests/MVVMDemoNew.Tests.csproj (hunk 1 above)
Comment (xcarruth) [reply MVVMDemoNew.Tests/MVVMDemoNew.Tests.csproj:11]: I dont know what this means

**Review by AECOM-CarruthX — APPROVED** [review]
Looks good now

**Pull request merged**